main.go

```diff
@@ -39,7 +39,27 @@ import (
 	"github.com/gatekeeper/gatekeeper-operator/pkg/platform"
 	"github.com/gatekeeper/gatekeeper-operator/pkg/util"
 	"github.com/gatekeeper/gatekeeper-operator/pkg/version"
+	constraintV1 "github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1beta1"
+	"github.com/open-policy-agent/gatekeeper/v3/apis/config/v1alpha1"
+	"github.com/open-policy-agent/gatekeeper/v3/apis/status/v1beta1"
```

Comment:
It makes it clear it's coming from Gatekeeper and not the operator CRs.
```suggestion
	gkv1alpha1 "github.com/open-policy-agent/gatekeeper/v3/apis/config/v1alpha1"
	gkv1beta1 "github.com/open-policy-agent/gatekeeper/v3/apis/status/v1beta1"
```